configure.ac

```diff
@@ -107,7 +108,17 @@ show_section_title "Configure build tools"
 
 # Find compiler, libtools, etc
 #
-LT_INIT([disable-static])
+AC_ARG_ENABLE([xlio-static-lib],
```

Comment:
I'd prefer to use --enable-static which comes out of box with LT_INIT and does the same logic. In this case LT_INIT([disable-static]) should be untouched and user may request --enable-static manually. We can pass this information further to c++ code via define to handle syscall symbols.

Reply:
Please suggest a fix.

Reply:
Done